base/experiments/ad9910_ram.py

```diff
@@ -0,0 +1,206 @@
+from artiq.experiment import *
+from artiq.coredevice.ad9910 import *
+from jax import JaxExperiment
+import numpy as np
+from enum import Enum
+
+
+__all__ = ["AD9910RAM"]
+
+
+class RAMType(Enum):
+    """The type of data in the RAM.
+
+    Enums:
+        FREQ: Data are treated as frequencies (Hz).
+            The exact frequencies will be played-back.
+        PHASE: Data are treated as phases (turns)
+            Phase modulation will be performed.
+        AMP: Data are treated as amplitudes.
+            Amplitude modulation will be performed.
+        Polar: Data are treated as amplitudes & phase.
+            Polar modulation will be performed.
+    """
+    FREQ = 1
+    PHASE = 2
+    AMP = 3
+    POLAR = 4
+
+
+class RAMProfile:
+    RAM_SIZE = 1024
```

Comment:
Is this a hard limit by ad9910?

Reply:
If it is a hard limit, this is a good number to know. We can calculate the theoretical quantum gate efficiency due to this RAM size, and determine if we need DRG for realizing certain gates of reasonable effciencies.

Reply:
> Is this a hard limit by ad9910?

Yes. From the datasheet: "The AD9910 makes use of a 1024 × 32-bit RAM." Each entry is a 32-bits word.

Reply:
And FYI, the RAM is shared between all DDS profiles. It doesn't matter if we only ever load 1 RAM profile per ad9910 at a time.

Reply:
Okay. Can you add these information in the docstring too?